frontend/lang/en.json

```diff
@@ -0,0 +1,47 @@
+{
+    "FormField": {
+        "Required": "Required",
+        "Loading": "Loading..."
+    },
+    "LocaleSelector": {
+        "Availabile In": "This site is available in:",
+        "Bottom Right": "You can change the language at any time in the bottom right corner of the page. ↘️"
+    },
+    "ModalError": {
+        "Heading": "An error occurred",
+        "Subheading": "Sorry about that, our team take bug reports seriously, and will try to make it right!",
+        "Error Details": "Error Details", 
+        "Common Steps": "",
```

Comment:
Should this have some copy? Won't it just be blank otherwise? On the other hand, I don't see it used anywhere.

Reply:
Good catch - fixed.